webpack/assets/javascripts/react_app/API.js

```diff
@@ -1,6 +1,18 @@
 import $ from 'jquery';
+import axios from 'axios';
+
+const onSuccess = response => response.data;
+const onError = error => Promise.reject(error);
 
 export default {
+  ajax(url) {
```

Comment:
why expose an "ajax" endpoint? (instead of replacing get, post put etc?)

Reply:
ajax is a temporarily endpoint, i'll merge it with get soon